test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -35,14 +35,6 @@ const skippedAuthTests = [
   'unset works with an encrypted field',
   'updateOne with deterministic encryption',
   'updateMany with deterministic encryption',
-  'type=date',
-  'type=regex',
-  'type=timestamp',
-  'type=javascript',
-  'type=binData',
-  'type=int',
-  'type=objectId',
-  'type=symbol',
```

Comment:
One of them failed (`type=javascript`) because our legacy test runner was strict about properties not existing as opposed to nullish. Code now always defines the property scope, but it is set to null when there is no scope, so that's the small test adjustment with the special case for 'Code'